package.json

```
@@ -132,7 +132,8 @@
     "tronb:package:prod": "npm run tronb:vite:prod && electron-builder --config electron-builder.yml --publish always",
     "test-setup": "npm install && npm run build:wasm",
     "test": "vitest --mode development",
-    "test:snapshots": "PLATFORM=web NODE_ENV=development playwright test --config=playwright.config.ts --grep=@snapshot --trace=on --shard=1/1",
+    "test:snapshots": "PLATFORM=web playwright test --config=playwright.config.ts --grep=@snapshot --trace=on --shard=1/1",
+    "test:snapshots:local": "PLATFORM=web NODE_ENV=development playwright test --config=playwright.config.ts --grep=@snapshot --trace=on --shard=1/1",
```

Assuming we actually need to set `NODE_ENV` at all, would this be more maintainable?

```suggestion
    "test:snapshots:local": "NODE_ENV=development npm run test:snapshots",
```

---

You're super right, thanks I'll commit this once the first run completes

---

2 failures, but very different failures from the other branch, this is promising!